src/Jackett.Common/Indexers/Abstract/GazelleTracker.cs

```diff
@@ -231,11 +231,9 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
 
                     if (imdbInTags)
                     {
-                        var imdbTags = tags
-                            .Select(tag => ParseUtil.GetImdbID((string)tag))
-                            .Where(tag => tag != null);
-                        if (imdbTags.Count() == 1)
-                            release.Imdb = imdbTags.First();
+                        release.Imdb = tags
+                                       .Select(tag => ParseUtil.GetImdbID((string)tag))
+                                       .Where(tag => tag != null).FirstIfSingleOrDefault();
```

Comment:
Is default null in this case? this is not clear to me, I prefer the old code in this case.

Reply:
Default is the same value as what it would be in `FirstOrDefault()` which is what the name is based off of. I offered an override for the default value so we could avoid all the other cases looking like `value.FirstIfSingleOrDefault() ?? "0"`

Reply:
I know, the problem with `FirstOrDefault()` is the definition itself in C# => `If a collection is empty, it returns the default value for the type.` That is a bad design, in other languages there is not such thing as "default". You have `GetOrElse(value)`, `FirstOrElse(value)`.... but never default.
When I'm reading the code I have to guess the type (because we don't have explicit types => var) and then I have to guess what will be the default value for that type.
Said that, I don't care as long as the code works as expected.

Reply:
For most values `default(T) == null` except `struct` and built in non-nullable types (`int float double` etc which are technically struct as well). I would assume `default` is equivalent to the uninitialized values of other languages (what is the value of i in `int i;` or str in `string str;` in any other language?) Or maybe I'm just that used to C#.

I will say that C# is fairly difficult to read outside of an IDE. Most C#/.Net IDEs feature things like type hinting and Intellisense (auto-complete), and at times I take for granted that I can review PRs from within VS2019 as a native feature so I get the type hinting the entire time.

Reply:
> what is the value of i in int i; or str in string str; in any other language?

Well that question requires a huge answer but I give some examples instead.
In Java you can use primitive types (int, char, long...) that will have 0 as default value because they can't be null. Or you can use Objects (Integer, Long, String...) that are null by default. It's recommended to initialize the var in the declaration to avoid confusion.
In C/C++ the default value will be random (the current state of the reserved memory) because C doesn't have default values, you are in charge of that.
In Python and many other languages you must initialize the var with some value, you can't just declare.

Reply:
I was pretty sure C++ also followed your Java description, but it's been a number of years since the last time I wrote anything in C++, so I could be wrong. But, with your examples, C# and the `default` keyword follows Java above. Anything that can't be null is 0.